pandas/_libs/tslibs/offsets.pyx

```diff
@@ -4086,15 +3958,15 @@ cdef inline int get_day_of_month(npy_datetimestruct* dts, day_opt) nogil except?
     cdef:
         int days_in_month
 
-    if day_opt == 'start':
+    if day_opt == "start":
```

Comment:
maybe want to make these an enum

Reply:
yah i looked at this, there are a couple of usages that dont follow this pattern yet